cloudify_cli/commands/profiles.py

```diff
@@ -189,13 +190,20 @@ def use(manager_ip,
             skip_credentials_validation=skip_credentials_validation,
             logger=logger,
             **kwargs)
-    cluster = env.profile.cluster.get(CloudifyNodeType.MANAGER)
-    if cluster:
-        client = env.get_rest_client(
-            client_profile=env.profile,
-            username=(env.profile.manager_username or 'admin'),
-            password=(env.profile.manager_password or 'admin')
-        )
+
+    active_profile = _get_profile(env.get_active_profile())
+    active_profile_context = env.get_profile_context(env.get_active_profile())
+    if active_profile_context.manager_username is None:     # docl 1st run
```

Comment:
what do u mean here?

Reply:
You can write the same as `if **not** active_profile_context.manager_username:`

Reply:
This function runs automatically as part of docl run, in which case there's no username nor password. L196 is for that case